src/lib/helpers/deployment.helper.ts

```diff
@@ -288,3 +302,25 @@ export function waitForBatchedPendingTransactions(
 
   return concat(transactions$, receipts$);
 }
+
+export function getContractAddressFromReceipt(
+  receipt: ethers.providers.TransactionReceipt,
+  isSignerUniversalProfile: boolean
+) {
+  return isSignerUniversalProfile
+    ? receipt.contractAddress ||
+        ethers.utils.defaultAbiCoder.decode(['address'], receipt.logs[1].topics[2]).toString()
+    : receipt.contractAddress || receipt.to;
+}
```

Comment:
I find this ternary a bit tricky to understand 
Is it saying : 
```
if (isSignerUniversalProfile == true)
{return 
receipt.contractAddress || ethers.utils.defaultAbiCoder.decode(['address'], receipt.logs[1].topics[2]).toString()}
else {
    return receipt.contractAddress || receipt.to;
}

```
I dont understand why in both cases we return `receipt.contractAddress`

What is the meaning of `|| `? Is it `or` ? for example for line 313 : if `receipt.contractAddress `not defined then return ` receipt.to` ?

Thanks

Reply:
yep || means or so is being used as fallback values here